### DISCUSSION TITLE
EULA about Leak source code

### DISCUSSION THREAD

**Discussion by hongphu8790** [dashingsoft/pyarmor#1351] [Q&A]
Our team decided to upgrade license, I have some questions for you, feel free to response to us:

I have followed this EULA: https://github.com/dashingsoft/pyarmor/blob/master/LICENSE 

Read through, i don't see anything about commitments about: stealing/leak source code from your company (imagine that your company has our product name, we distribute obfuscated script, your company may potentially use our product name to search, use private capsule and decrypt  obfuscated script, you are not, but some colleagues or another system security issue). I'm worried about that, have commitments there?

**Comment by jondy** [comment]
Please check item `3. Responsibilities and Commitments`
